packages/sdk/test/cross-chain-provider.spec.ts

```diff
@@ -783,46 +760,204 @@ describe('CrossChainProvider', () => {
+          )
+
+          await submitStateRootBatchForMessage(message)
+
+          expect(await provider.getMessageStatus(message)).to.equal(
+            MessageStatus.IN_CHALLENGE_PERIOD
+          )
+        })
       })
 
       describe('when the message is no longer in the challenge period', () => {
         describe('when the message has been relayed successfully', () => {
-          it('should return a status of RELAYED')
+          it('should return a status of RELAYED', async () => {
+            const message = await sendAndGetDummyMessage(
+              MessageDirection.L2_TO_L1
+            )
+
+            await submitStateRootBatchForMessage(message)
+
+            const challengePeriod = await provider.getChallengePeriodBlocks()
+            for (let x = 0; x < challengePeriod + 1; x++) {
+              await ethers.provider.send('evm_mine', [])
+            }
+
+            await l1Messenger.triggerRelayedMessageEvents([
+              hashCrossChainMessage(message),
+            ])
+
+            expect(await provider.getMessageStatus(message)).to.equal(
+              MessageStatus.RELAYED
+            )
+          })
         })
 
         describe('when the message has been relayed but the relay failed', () => {
-          it('should return a status of READY_FOR_RELAY')
+          it('should return a status of READY_FOR_RELAY', async () => {
+            const message = await sendAndGetDummyMessage(
+              MessageDirection.L2_TO_L1
+            )
+
+            await submitStateRootBatchForMessage(message)
+
+            const challengePeriod = await provider.getChallengePeriodBlocks()
+            for (let x = 0; x < challengePeriod + 1; x++) {
+              await ethers.provider.send('evm_mine', [])
+            }
+
+            await l1Messenger.triggerFailedRelayedMessageEvents([
+              hashCrossChainMessage(message),
+            ])
+
+            expect(await provider.getMessageStatus(message)).to.equal(
+              MessageStatus.READY_FOR_RELAY
+            )
+          })
         })
 
         describe('when the message has not been relayed', () => {
-          it('should return a status of READY_FOR_RELAY')
+          it('should return a status of READY_FOR_RELAY', async () => {
+            const message = await sendAndGetDummyMessage(
+              MessageDirection.L2_TO_L1
+            )
+
+            await submitStateRootBatchForMessage(message)
+
+            const challengePeriod = await provider.getChallengePeriodBlocks()
+            for (let x = 0; x < challengePeriod + 1; x++) {
+              await ethers.provider.send('evm_mine', [])
+            }
+
+            expect(await provider.getMessageStatus(message)).to.equal(
+              MessageStatus.READY_FOR_RELAY
+            )
+          })
         })
       })
     })
 
     describe('when the message does not exist', () => {
-      it('should throw an error')
+      it('should throw an error', async () => {
```

Comment:
I feel like it should not throw an error but instead return `null`. Can you `skip` this test to make sure we know its not implemented when running the tests?